webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue

```diff
@@ -128,25 +138,6 @@
         v-model="news.attachments"
         @HideAttachmentsDrawer="onHideAttachmentsDrawer"
         @uploadingCountChanged="setUploadingCount" />
-      <!-- The following bloc is needed in order to display the pin confirmation popup -->
```

Comment:
we need also to clean the other references to publish/unpublish except publishing drawer